src/app/helptext/system/certificates.ts

```diff
@@ -255,7 +252,16 @@ key used to sign this certificate.'),
       },
       enabled: {
         placeholder: T('Authority Key Identifier'),
-        tooltip: T('Activate this certificate extension.'),
+        tooltip: T('Activate this extension.\
```

Comment:
I recommend:

`Activate this extension to identify the public key corresponding to the private key that signed a certificate. This extension is useful when an issuer has multiple signing keys (either due to multiple concurrent key pairs or due to changeover). Authority Key Identifier MAY base the identification on either the key identifier (the subject key identifier in the issuer's certificate) or the issuer's name and serial number.`